src/defaultSettings.js

```diff
@@ -13,7 +13,7 @@ export const rendererSpecificSettings = {
     details: null,
     cutoff: 10,
     alphabetize: false,
-    exportable: false
+    exportable: true
```

Comment:
Update the settings schema accordingly, and also expose the [`nRowsPerPage`](https://github.com/RhoInc/webcharts/wiki/Table-Configuration#nrowsperpage) setting because I could see the DMs wanting to embiggen this baby.